src/test/db.rs

```diff
@@ -318,3 +323,61 @@ async fn db_aggregate_disk_use() {
         .await
         .expect("aggregate with disk use should succeed");
 }
+
+#[cfg_attr(feature = "tokio-runtime", tokio::test)]
+#[cfg_attr(feature = "async-std-runtime", async_std::test)]
+async fn create_index_options_defaults() {
+    let defaults = IndexOptionDefaults {
+        storage_engine: doc! { "wiredTiger": doc! {} },
+    };
+    index_option_defaults_test(Some(defaults)).await;
+}
+
+#[cfg_attr(feature = "tokio-runtime", tokio::test)]
+#[cfg_attr(feature = "async-std-runtime", async_std::test)]
+async fn create_index_options_defaults_not_specified() {
+    index_option_defaults_test(None).await;
+}
+
+#[function_name::named]
```

Comment:
I think the two individually tests should pass in their function names to use rather than using the name of the shared function here. Otherwise, since the tests are running concurrently, they could race to create the same collection with different options.